datagateway_api/src/common/filters.py

```diff
@@ -27,11 +27,23 @@ def __init__(self, field, value, operation):
         self.value = value
         self.operation = operation
 
-        if self.operation == "in":
+        if self.operation in ["in", "nin", "inq"]:
             if not isinstance(self.value, list):
                 raise BadRequestError(
-                    "When using the 'in' operation for a WHERE filter, the values must"
-                    " be in a list format e.g. [1, 2, 3]",
+                    f"When using the {self.operation} operation for a WHERE filter, the"
+                    f" values must be in a list format e.g. [1, 2, 3]",
+                )
+
+        if self.operation == "between":
+            if not isinstance(self.value, list):
+                raise BadRequestError(
+                    "When using the 'between' operation for a WHERE filter, the values"
+                    " must be in a list format e.g. [1, 2]",
+                )
```

Comment:
This looks like duplicated code. Could you add `"between"` to the list on L30 and remove this duplicated code here?

Reply:
That will work, thanks.